### ISSUE TITLE
`ci.yaml validation` error: "Resource was Not Found"

### ISSUE THREAD

**Issue by gaaclarke** [flutter/flutter#87965]
**Description**
`ci.yaml validation` is failing on an unrelated change blocking the dart->flutter engine roller

**Error**
GitHub Error: Requested Resource was Not Found

**Example**
https://github.com/flutter/engine/pull/27965/checks?check_run_id=3283405204

**Comment by keyonghan** [comment]
Sounds a duplicate of https://github.com/flutter/flutter/issues/85483
/cc @CaseyHillers

**Comment by CaseyHillers** [comment]
This is a unique issue.

I've been able to reproduce this by force-pushing commits. Pushing an empty will unblock you in these cases.

I'll investigate why Cocoon is not getting the new ref

**Comment by CaseyHillers** [comment]
https://github.com/flutter/engine/pull/28830 is an instance where it wasn't force pushed
